db/migrate/20170417192246_create_interactions_publications_table.rb

```diff
@@ -0,0 +1,15 @@
+class CreateInteractionsPublicationsTable < ActiveRecord::Migration
+	def up
+		create_table :interactions_publications, id: false do |t|
+			t.text :interaction_id, null: false
+			t.text :publication_id, null: false
+		end
+		execute 'ALTER TABLE interactions_publications ADD PRIMARY KEY (interaction_id, publication_id)'
+    	execute 'ALTER TABLE interactions_publications ADD CONSTRAINT fk_interaction FOREIGN KEY (interaction_id) REFERENCES interactions(id)'
```

Comment:
This line and the one below should be indented to the same level as line 7 above.